### ISSUE TITLE
[ Nushell ] Deprecation notice for future versions

### ISSUE THREAD

**Issue by Cyanide-1** [ajeetdsouza/zoxide#654]
I just installed zoxide for the first time on my pc and set it up in nushell according to the exact instructions given, this is a very vanilla shell so nothing should be bothering zoxide or conflicting with it.

Every time i launch the shell i get this error in my terminal: 
![Screenshot from 2023-12-06 07-30-58](https://github.com/ajeetdsouza/zoxide/assets/136278181/5850613a-0030-437c-843a-cd6b20294add)

Zoxide seems to still work, but there is a deprecation notice stated for the next version of nushell, would like this to be addressed to the devs of this project, thank you for your time and i hope you have a lovely day whoever is reading this.

**Comment by CarterLi** [comment]
How about a new release?

**Comment by lexpierce** [comment]
```
> nu
Error: nu::shell::external_command

  × External command failed
    ╭─[/Users/lexpierce/.cache/zoxide/init.nu:25:1]
 25 │ # Jump to a directory using only keywords.
 26 │ def-env __zoxide_z [...rest:string] {
    · ───┬───
    ·    ╰── did you mean 'let-env'?
 27 │   let arg0 = ($rest | append '~').0
    ╰────
  help: No such file or directory (os error 2)
```

Today is the day it broke. Can we get a new release? nu 0.88 is now released.

**Comment by acidghost** [comment]
@ajeetdsouza ^

**Comment by CarterLi** [comment]
In case anyone needs, I compiled zoxide for windows with the latest code source

https://github.com/CarterLi/zoxide/releases/tag/nufix

**Comment by grindarius** [comment]
> while we wait for @ajeetdsouza to release, is there an easy way to solve this on macos by editing some file?

You can install from git for now by running `cargo install zoxide --git https://ajeetdsouza/zoxide`

**Comment by malfroid** [comment]
Thanks!

I first uninstalled zoxide with `brew uninstall zoxide`.
To install the latest zoxide from this repo, I needed a slightly different command including `github.com/`: `cargo install zoxide --git https://github.com/ajeetdsouza/zoxide`
Then I needed to add `~/.cargo/bin` to my PATH: https://www.nushell.sh/cookbook/setup.html#configure-your-path-and-other-environment-variables
Finally the error disappeared.

**Comment by IvanProg00** [comment]
It's good, that exists solution via `cargo`, but I would like to use **Brew** package manager for terminal tools like `zoxide`. When will be released the new version of `zoxide`?

**Comment by MAST1999** [comment]
For anyone else waiting, you can change `dev-env`  to `dev --env` and it'll work.

Just make sure that you don't re-create the file each time you open `nu` or your changes will be overridden.

**Comment by ajeetdsouza** [comment]
Hey, so sorry about the delayed responses. I've been out of town the last couple of months, and wasn't able to check GitHub.

I've released [v0.9.3](https://github.com/ajeetdsouza/zoxide/releases/tag/v0.9.3), it should fix all the breaking changes in Nushell 0.89.0.

Thanks for your patience!
